hnn_core/network_models.py

```diff
@@ -174,7 +174,7 @@ def jones_2009_model(params=None, add_drives_from_params=False,
 
 
 def law_2021_model(params=None, add_drives_from_params=False,
-                   legacy_mode=True):
+                   legacy_mode=False):
```

Comment:
wondering if we should be annoying about this and already add a [deprecation warning](https://docs.python.org/3/library/exceptions.html#DeprecationWarning) if `legacy_mode=True` ... so we don't get tied to the ghosts of the past.